### PR TITLE
Upload files via API

### LINKED ISSUE #62: Add example about how to upload samples through API
* Sample uploading through API
* The full context of possible usage (uploading samples + collecting results)

(reported on Twitter: https://twitter.com/matte_lodi/status/1255080965556391936 -> reply when ticket is resolved)

### PR DESCRIPTION
**Pull request by chivay**
resolve #62

### CODE REVIEW

File: drakcore/drakcore/frontend/src/UploadSample.js [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
+    this.onFileChange = this.onFileChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  onFileChange(event) {
+    event.preventDefault();
+    this.setState({ file: event.target.files[0] });
+  }
+
+  onSubmit(event) {
```
Comment (BonusPlay) [inline drakcore/drakcore/frontend/src/UploadSample.js:22]: async/await 😢

File: drakcore/drakcore/frontend/src/UploadSample.js [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+    event.preventDefault();
+    if (this.state.file === null) return;
+    api.uploadSample(this.state.file).then((response) => {
```
Comment (BonusPlay) [inline drakcore/drakcore/frontend/src/UploadSample.js:25]: async/await 😢

Comment (chivay) [reply drakcore/drakcore/frontend/src/UploadSample.js:25]: oh yes

**Comment by chivay** [comment]
> cool! could you also add an example python script in `examples/`? such API docs were requested in community feedback

Sure, I'll add it.

**Pull request merged**